ONSdigital/response-operations-ui
# Refactor templates

# Motivation and Context
Response Operations templates have a structure that isn't quickly or easily understood, increase clarity and improve structure.  Reduce code reuse.

# What has changed
* Several elements have been extracted into patterns for reuse
* Several of the more problematic templates have been refactored

# How to test?
Run response ops, observe functioning, with focus on avoiding regressions
Ensure acceptance tests pass.

# Links
https://trello.com/c/AJhShGha
